codeforboston/maple
# dropdown

# Summary

#1307

Added dynamic dropdown component with customizable variant, width and height
Open-ended customization is in alignment with open-ended state of UX library
Text wrap for smaller devices

# Checklist

- [X ] If I've added shared components, I've added a storybook story.
- [ X] I've made pages responsive and look good on mobile.

# Screenshots

[_Add some screenshots highlighting your changes._](https://github.com/codeforboston/maple/assets/101157878/9c18fdd2-7f55-4d76-bb85-654e915bbb50)


# Steps to test/reproduce

_For each feature or bug fix, create a step by step list for how a reviewer can test it out. E.g.:_

1. Go to the faq page
1. Click on dropdown items
1. See that it filters content
